lib/hammer_cli_foreman/attribute.rb

```diff
@@ -42,8 +42,14 @@ class Create < HammerCLIForeman::CreateCommand
 
       def request_params
         params = super
+        compute_resource_name = HammerCLIForeman.record_to_common_format(
+            HammerCLIForeman.foreman_resource(:compute_resources).call(:show, 'id' => option_compute_resource_id )
+        )["name"]
+        interfaces_attrs_name_list = {"libvirt" => "nics_attributes","gce" => "network_interfaces_nics_attributes"}
```

Comment:
This info is feature of compute resource, shouldn't it be in the appropriate class?
e.g. `HammerCLIForeman.compute_resources[compute_resource_name].try('interfaces_attr_name')`

Reply:
It could set in the `Base` to "interfaces_attributes" and over-ridden in resources where needed. As it is now there is no way to easily add new resource with non-standard nics naming